.editorconfig

```diff
@@ -119,6 +119,9 @@ csharp_prefer_simple_default_expression = true:suggestion
 
 dotnet_code_quality_unused_parameters = non_public:suggestion
 
+# https://learn.microsoft.com/en-gb/dotnet/fundamentals/code-analysis/quality-rules/ca1859
+dotnet_diagnostic.CA1859.severity = suggestion
+
```

Comment:
@iSazonov We could probably move all our existing rules from globalconfig here. Not sure if diagnostics deserves its only section in editorconfig though.

Reply:
Added in 98579797f